src/main/resources/assets/powersuits/lang/zh_CN.lang

```diff
@@ -135,13 +140,29 @@ gui.install=安装
 gui.salvage=回收
 
 gui.newKeybind=新建
-gui.trashKeybind=删除!
+gui.trashKeybind=删除
 gui.pressKey=按键
 gui.keybindInstructions1=使用"新建"来绑定新快捷键.
 gui.keybindInstructions2=拖拽模块完成绑定.
 gui.keybindInstructions3=拖到"删除!"上解除绑定.
```

Comment:
line 143 has removed the exclamation mark; thus here should do the same thing.
(Using a computer which cannot type Chinese temporarily. I apologize for that.)